solidus.gemspec

```diff
@@ -26,5 +26,5 @@ Gem::Specification.new do |s|
   s.add_dependency 'solidus_core', s.version
   s.add_dependency 'solidus_sample', s.version
 
-  s.add_dependency 'solidus_frontend', '~> 3.4.0.dev'
+  s.add_dependency 'solidus_frontend', '~> 4.0.0.dev'
```

Comment:
@waiting-for-dev is this updated by the post-release workflow, or did you change it manually? If the former, we should also remove that bumper that in https://github.com/solidusio/solidus/pull/5026. cc @elia

Reply:
@kennyadsl, I did it manually. It'd have been cleaner doing that on a separate commit.